dm/worker/server.go

```diff
@@ -640,21 +640,23 @@ func (s *Server) startWorker(cfg *config.MysqlConfig) error {
 
 	ectx, cancel := context.WithTimeout(s.etcdClient.Ctx(), time.Second*3)
 	defer cancel()
-	resp, err := s.etcdClient.Get(ectx, common.UpstreamSubTaskKeyAdapter.Encode(cfg.SourceID))
-	if err == nil {
-		for _, kv := range resp.Kvs {
-			infos := common.UpstreamSubTaskKeyAdapter.Decode(string(kv.Key))
-			taskName := infos[1]
-			task := string(kv.Value)
-			cfg := config.NewSubTaskConfig()
-			if err := cfg.Decode(task); err != nil {
-				return nil
-			}
-			if err := w.StartSubTask(cfg); err != nil {
-				return nil
-			}
-			log.L().Info("load subtask successful", zap.String("sourceID", cfg.SourceID), zap.String("name", taskName))
+	key := common.UpstreamSubTaskKeyAdapter.Encode(cfg.SourceID)
+	resp, err := s.etcdClient.KV.Get(ectx, key, clientv3.WithPrefix())
+	if err != nil {
+		return err
+	}
+	for _, kv := range resp.Kvs {
+		infos := common.UpstreamSubTaskKeyAdapter.Decode(string(kv.Key))
+		taskName := infos[1]
+		task := string(kv.Value)
+		cfg := config.NewSubTaskConfig()
+		if err := cfg.Decode(task); err != nil {
+			return nil
+		}
+		if err := w.StartSubTask(cfg); err != nil {
+			return nil
 		}
+		log.L().Info("load subtask successful", zap.String("sourceID", cfg.SourceID), zap.String("name", taskName))
```

Comment:
```suggestion
		log.L().Info("load subtask successful", zap.String("sourceID", cfg.SourceID), zap.String("task", taskName))
```

Is `task` more meaningful here? just as we did in _syncer.go_. `tcontext.Background().WithLogger(log.With(zap.String("task", cfg.Name), zap.String("unit", "binlog replication")))`.